prisma/seed.ts

```diff
@@ -152,9 +169,13 @@ const createExercises = async () => {
 };
 
 const main = async () => {
+  await prisma.activity.deleteMany();
```

Comment:
künnt ma dia calls ned in die jeweilige funktion schiaba. denn künnt ma `main` clean bheba.